WasatchNET/Spectrometer.cs

```diff
@@ -1543,11 +1646,65 @@ public virtual byte laserTemperatureSetpointRaw
                 if (haveCache(op) && value == laserTemperatureSetpointRaw_)
                     return;
 
-                sendCmd(Opcodes.SET_LASER_TEC_SETPOINT, laserTemperatureSetpointRaw_ = Math.Min((byte)127, value));
+                sendCmd(Opcodes.SET_LASER_TEC_SETPOINT, laserTemperatureSetpointRaw_ =  value);
+                readOnce.Add(op);
+            }
+        }
+        protected ushort laserTemperatureSetpointRaw_;
+
+        public virtual bool laserTECEnabled
+        {
+            get
+            {
+                if (!eeprom.hasLaser)
+                    return false;
+
+                if (laserTECMode == 0)
+                    return false;
+                else
+                    return true;
+            }
+            set
+            {
+                if (!eeprom.hasLaser)
+                    return;
+
+                if (value)
+                    laserTECMode = 1;
+                else
+                    laserTECMode = 0;

```

Comment:
The point of declaring an enum is to use the labels rather than the integral positions :-)

Reply:
agreed and fixed